text/game/abilities.stringtable

```diff
@@ -1406,7 +1406,7 @@
     </Entry>
     <Entry>
       <ID>294</ID>
-      <DefaultText>Schwingt die Nahkampfwaffe des Kämpfers in einem großen Bogen und fügt Feinden den Niederschlag Effekt zu.</DefaultText>
+      <DefaultText>Schwingt die Nahkampfwaffe des Kämpfers in einem großen Bogen und fügt Feinden den Niederstrecken Effekt zu.</DefaultText>
```

Comment:
Grade gesehen, in der Beschreibung von Rücksichtsloser Angriff wird Nahkampfgenauigkeit auf Genauigkeit verlinkt, und ebenso Nahkampfschaden auf Schaden. Groß und Kleinschreibung scheint dafür also egal zu sein.